app/server/lib/AttachmentFileManager.ts

```diff
@@ -218,3 +460,22 @@ export class AttachmentFileManager implements IAttachmentFileManager {
     };
   }
 }
+
+async function validateFileChecksum(fileIdent: string, fileData: Buffer): Promise<boolean> {
+  return fileIdent.startsWith(await checksumFileStream(Readable.from(fileData)));
+}
+
+interface AttachmentFileManagerLogInfo {
+  fileIdent?: string;
+  storeId?: string | null;
+}
+
+interface AttachmentFileInfo {
+  ident: string;
+  storageId: string | null;
+  data: Buffer;
+}
+
+interface TransferJob {
```

Comment:
What is the value of this interface compared to just a Promise? Could it just be `type TransferJob = Promise<void>`?

Reply:
Same answer as above :)